src/mb_caa_dimensions/image.ts

```diff
@@ -110,7 +110,7 @@ function urlToCacheKey(fullSizeUrl: string, thumbnailUrl?: string): string {
     // Ideally, the cache key for RG covers would be the full size URL of the release cover,
     // but we unfortunately cannot get the original image's extension here, so we cannot construct
     // it.
-    if (urlObject.host === CAA_DOMAIN && urlObject.pathname.startsWith('/release-group/')) {
+    if (AA_DOMAINS.test(urlObject.host) && urlObject.pathname.startsWith('/release-group/')) {
```

Comment:
Since this code branch only deals with release group cover art, you can revert this change (and restore the CAA_DOMAIN constant in addition to the regex).

Reply:
Why not.
But that would make two constants (`CAA_DOMAIN` and `AA_DOMAINS`\*) instead of one.
What do you think @ROpdebee?

\* `AA_DOMAINS`should probably become `IMG_DOMAINS`, see https://github.com/ROpdebee/mb-userscripts/pull/779#pullrequestreview-2190285233
`CAA` project was renamed to `IMG` in https://tickets.metabrainz.org where `IMG = CAA + EAA` 😉

Reply:
I made this change and also renamed my made-up AA into IMG, like in https://tickets.metabrainz.org/projects/IMG to answer [your P.S.](https://github.com/ROpdebee/mb-userscripts/pull/779#pullrequestreview-2190285233).
Please check.

I didn't manage tu build (`npm run build` failed). :)
There is a kind of install step before, like `npm install --save-dev` or something like that?